integrationtests/chrome_test.go

```diff
@@ -45,7 +45,7 @@ func init() {
 var _ = Describe("Chrome tests", func() {
 	It("does not work with mismatching versions", func() {
 		versionForUs := protocol.SupportedVersions[0]
-		versionForChrome := protocol.SupportedVersions[len(protocol.SupportedVersions)-1]
+		versionForChrome := protocol.SupportedVersions[1]
 
 		// If both are equal, this test doesn't make any sense.
 		if versionForChrome == versionForUs {
```

Comment:
I was hoping that the above change is only temporary and we can remove it either by updating the Docker image or merging your Chrome-branch.